lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -3076,12 +3089,31 @@ def model_add_construction(model, construction_name, construction_props = nil, s
       return OpenStudio::Model::OptionalConstruction.new
     end
 
+    intended_surface_type = data["intended_surface_type"]
+    intended_surface_type ||= ''
+
     # Make a new construction and set the standards details
-    if data['intended_surface_type'] == 'GroundContactFloor' && !surface.nil?
-      construction = OpenStudio::Model::FFactorGroundFloorConstruction.new(model)
-    elsif data['intended_surface_type'] == 'GroundContactWall' && !surface.nil?
-      construction = OpenStudio::Model::CFactorUndergroundWallConstruction.new(model)
-    else
+    is_layered_construction = true
+
+    if intended_surface_type == 'GroundContactFloor' && !surface.nil?
+      if construction_props
+        construction = OpenStudio::Model::FFactorGroundFloorConstruction.new(model)
+        is_layered_construction = false
+      else
+        OpenStudio.logFree(OpenStudio::Warn, 'openstudio.standards.Model', "Construction properties not specified for '#{construction_name}', cannot create F-Factor Ground Floor Construction.  A regular construction will be created instead, and Surface '#{surface.name}' will be set to use the 'Ground' outside boundary condition (previously '#{surface.outsideBoundaryCondition}').")
+        surface.setOutsideBoundaryCondition('Ground')
+      end
+    elsif intended_surface_type == 'GroundContactWall' && !surface.nil?
+      if construction_props
+        construction = OpenStudio::Model::CFactorUndergroundWallConstruction.new(model)
+        is_layered_construction = false
+      else
+        OpenStudio.logFree(OpenStudio::Warn, 'openstudio.standards.Model', "Construction properties not specified for '#{construction_name}', cannot create C-Factor Underground Wall Construction.  A regular construction will be created instead, and Surface '#{surface.name}' will be set to use the 'Ground' outside boundary condition (previously '#{surface.outsideBoundaryCondition}').")
+        surface.setOutsideBoundaryCondition('Ground')
+      end
+    end
+
+    if is_layered_construction
```

Comment:
Second error handling: This is #1 Pre conditions from https://github.com/NREL/openstudio-standards/issues/2009#issuecomment-3296652686

A FFactor/CFactor construction would be created even if construction_props was somehow nil, and it would have NOTHING set in it (defaulted), and surface if present would not have its outside boundary condition set properly.